data/hypervisor.args.in

```diff
@@ -2,7 +2,7 @@
 -name
 @NAME@
 -machine
-pc-lite,accel=kvm,kernel_irqchip,nvdimm
+pc,accel=kvm,kernel_irqchip,nvdimm
```

Comment:
This -- and the change to a compressed kernel -- are big changes which will impact performance and memory density so we would need a lot more detail on the cost/benefit of such a change before making this the default.

@anthonyzxu might want to comment on your commit message for https://github.com/01org/cc-oci-runtime/pull/787/commits/9676ed71990de8314794e71b38cecb240dbb9397.

Iff DPDK absolutely requires these values, like `@VM_MEM@` and `@VM_MEM_MAX@`, we'll have to make the runtime handle the switch to a different kernel type and hypervisor machine type.